src/commons/api/ca.rs

```diff
@@ -1590,6 +1600,13 @@ impl ChildConnectionStats {
     pub fn new(handle: ChildHandle, last_exchange: Option<ChildExchange>) -> Self {
         ChildConnectionStats { handle, last_exchange }
     }
+
+    pub fn inactive(&self, threshold_hours: i64) -> bool {
+        match &self.last_exchange {
+            None => false, // if there has been no exchange at all, the child is not yet active, rather than inactive
+            Some(exchange) => exchange.timestamp < (Time::now() - Duration::hours(threshold_hours)).timestamp(),
```

Comment:
This logic doesn't seem consistent with the documentation above that says that child CAs "republish manifests and CRLs 8 hours before they would expire" as the latter talks about how near to the moment of expiration we are while the code talks about how long it has been since there was an exchange with the client, perhaps the code better matches the documentation where it says "Krill CAs will send RFC 6492 list every 10 minutes"? (and is that a reference to Krill _child_ CAs?)